js/index-min.js

```diff
@@ -1,7 +1 @@
-var $=jQuery;$(document).ready(function(){$(window).scroll(function(){var distanceScrolled=$(this).scrollTop();if(distanceScrolled>60||$(".menu").hasClass('open')){$('.menu').css('background',"#697061")}});var $root=$('html, body');$('a.scroll-tag').click(function(){$(".menu.open").removeClass("open").addClass("closed");if($(this).attr('href')[0]=="#"){event.preventDefault()
-  $root.animate({scrollTop:$('[name="'+$.attr(this,'href').substr(1)+'"]').offset().top-80},450);return!1}});$('#copyright').on('click',function(){$('.image-block img').each((index,element)=>{$(element).attr('src',refineImage(element))})})
-  function refineImage(element){var imageSrc=$(element).attr('src')
-  var splitImg=imageSrc.split('.')
-  splitImg[0]=splitImg[0]+'-refined'
-  return splitImg.join('.')}
-  $(".menu").on("click",function(){if($(".menu").hasClass('closed')){$(".menu.closed").removeClass("closed").addClass("open")}else{$(".menu.open").removeClass("open").addClass("closed")}});$('#rsvp-form').on('submit',function(){$(this).css('opacity',0);$('#form-submitted').css('opacity',1)})})
+var $=jQuery;$(document).ready(function(){$(window).scroll(function(){($(this).scrollTop()>60||$(".menu").hasClass("open"))&&$(".menu").css("background","#697061")});var o=$("html, body");$("a.scroll-tag").click(function(){if($(".menu.open").removeClass("open").addClass("closed"),"#"==$(this).attr("href")[0])return event.preventDefault(),o.animate({scrollTop:$('[name="'+$.attr(this,"href").substr(1)+'"]').offset().top-80},450),!1}),$("#copyright").on("click",function(){$(".image-block img").each((o,s)=>{$(s).attr("src",function(o){var s=$(o).attr("src").split(".");return s[0]=s[0]+"-refined",s.join(".")}(s))})});var s=0;$("#gocrazy").on("click",function(){5==s&&($(".image-block img").each((o,s)=>{$(s).attr("src","images/gocrazy.gif")}),$("#gocrazy").append('<audio loop autoplay><source src="sounds/gocrazy.mp3" type="audio/mpeg"></audio>')),s++}),$(".menu").on("click",function(){$(".menu").hasClass("closed")?$(".menu.closed").removeClass("closed").addClass("open"):$(".menu.open").removeClass("open").addClass("closed")}),$("#rsvp-form").on("submit",function(){$(this).css("opacity",0),$("#form-submitted").css("opacity",1)})});
```

Comment:
Can you please change the `s` & `o`'s to readable variable names? I would prefer ease of reading over how slick the code looks.  Also, can we leave the `refineImage(element)` part. I think it makes it a little easier to understand what we are changing the image `src` too.  Lastly, can you change `5 == s` to `s == 5`, just OCD lol.

Reply:
whatever man this is bs